policyengine_us/parameters/gov/hhs/medicaid/eligibility/categories/pregnant/income_limit.yaml

```diff
@@ -45,6 +45,7 @@ KS:
   2021-01-01: 1.71
 KY:
   2021-01-01: 2.00
+  2023-01-01: 1.95
```

Comment:
Are you sure about these 5pp changes? There's a 5% income disregard and KFF isn't clear about when they include it or not.

Reply:
@vrathi101 could you check how we accounted for the limits prior - did we include the 5% disregard?

Reply:
Removed these to be consistent across parameter files